api-extractor/src/definitions/ApiItem.ts

```diff
@@ -6,6 +6,52 @@ import Extractor from '../Extractor';
 import ApiDocumentation from './ApiDocumentation';
 
 /**
+ * Indicates the type of definition represented by a ApiItem object.
+ */
+export enum ApiItemKind {
+  /**
+    * A TypeScript class.
+    */
+  class = 0,
```

Comment:
Our coding convention is InitialCaps for enum members.  This will also avoid any confusion with TypeScript language keywords such as "class"